packages/react-components/react-positioning/src/usePositioning.ts

```diff
@@ -173,12 +174,14 @@ function usePositioningOptions(options: PositioningOptions) {
   const { dir } = useFluent();
   const isRtl = dir === 'rtl';
   const positionStrategy: Strategy = strategy ?? positionFixed ? 'fixed' : 'absolute';
+  const normalizedAutoSize = normalizeAutoSize(autoSize);
```

Comment:
nit: I would flip this and call the user autoSize sth else, but that's just personal preference

Reply:
I have `const autoSize = normalizeAutoSize(rawAutoSize);`